### PR TITLE
Feature/3 add linter script

### PR DESCRIPTION
**Pull request by OlegMikus**
To keep code base clean, add linter to project.

Used packages:  mypy, pycodestyle, flake8, pylint

It is .sh script that runs linting for either all project files.

### CODE REVIEW

**Review by ishaurko — CHANGES_REQUESTED** [review]
exclude all files not related to linter

File: codestyle.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo ">>> $(basename ${BASH_SOURCE[0]})"
+
+set -o errexit # exit script when command fails
+set -o pipefail # this setting prevents errors in a pipeline from being masked
+set -o nounset # exit script when it tries to use undeclared variables
+
+echo 'pylint'
+pylint .
```
Comment (ishaurko) [inline codestyle.sh:10]: you can load flask plugin via
`pylint --load-plugins pylint_flask`

Comment (OlegMikus) [reply codestyle.sh]: Done

File: codestyle.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (OlegMikus) [reply codestyle.sh]: Done

**Pull request closed without merging**